derp/derp_server.go

```diff
@@ -49,6 +50,8 @@ import (
 // verbosely log whenever DERP drops a packet.
 var verboseDropKeys = map[key.NodePublic]bool{}
 
+var clock = tstime.StdClock{}
```

Comment:
this `clock` is of static type `tstime.StdClock` which means no other clock type can be assigned here.

I assume this should be:

```go
   var clock tstime.Clock = tstime.StdClock{}
```